docs/_posts/2020-01-09-new-year-new-bolt.md

```diff
@@ -0,0 +1,28 @@
+---
+title: New Year, New Bolt
+---
+
+Welcome to 2020!
+
+The team has been hard at work since the last update, adding a bevy of new features as well as formalizing and sanding the rough edges of existing features to prepare for the imminent Bolt 2.0 release.
```

Comment:
```suggestion
The team has been ~~on vacation~~ hard at work since the last update, adding a bevy of new features as well as formalizing and sanding the rough edges of existing features to prepare for the imminent Bolt 2.0 release.
```

My attempt at humor